app/services/gpo_daily_test_sender.rb

```diff
@@ -24,7 +27,7 @@ def otp_from_date(date = Time.zone.today)
 
   # @return [Hash]
   def designated_receiver_pii
-    @designated_receiver_pii ||= IdentityConfig.store.gpo_designated_receiver_pii
+    @designated_receiver_pii ||= (IdentityConfig.store.gpo_designated_receiver_pii || {})
```

Comment:
When would we expect this to be `nil` in the real-world? Just wondering about how we default configs; should we expect it to always be set from a default? Or have the `IdentityConfig` class handle a default?

Reply:
I don't think there's a way for it to be nil currently

Reply:
Oh right I though we had `allow_nil` in the config but we do not... will update

Reply:
simplified in 0f2e159cf8d1d6c742fa922e86950b179c3abc0b